modules/common/math/mpc_solver.h

```diff
@@ -53,7 +53,9 @@ bool SolveLinearMPC(
     const Eigen::MatrixXd &matrix_upper,
     const Eigen::MatrixXd &matrix_initial_state,
     const std::vector<Eigen::MatrixXd> &reference, const double eps,
-    const int max_iter, std::vector<Eigen::MatrixXd> *control);
+    const int max_iter, std::vector<Eigen::MatrixXd> *control,
+    std::vector<Eigen::MatrixXd> *control_gain,
```

Comment:
you could use "Matrix" as you already have "using Matrix = Eigen::MatrixXd;"

Reply:
not in .h file

Reply:
I am talking about .cc

Reply:
Hi, Jinyun. Thanks a lot for your comments. I have already fixed them accordingly.